CHANGELOG.md

```diff
@@ -18,6 +18,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## 📚 Documentation -->
 
+# [0.26.3] (unreleased) - 2024-12-dd
+
+## 🐛 Fixes
+
+- **`rover persisted-queries publish` supports `clientName` for operations - @glasser #2239**
+
+  If the JSON manifest file provided to `rover persisted-queries publish` sets the `clientName` on individual operations, these client names are included in the publish instead of ignored. Additionally, the new `--for-client-name` option overrides the `clientName` field for all operations in the manifest file.
```

Comment:
I may have missed this, but are there already docs PRs for:
1. The `--for-client-name` option in https://www.apollographql.com/docs/rover/commands/persisted-queries
2. Updating the manifest format with the `client-name` field in https://www.apollographql.com/docs/graphos/platform/security/persisted-queries#31-generate-persisted-query-manifests?

It would be great to include links to the relevant docs in the changelog entry.

Reply:
I'm planning to do full docs PRs once the feature is entirely enabled (Router v1.59.0 is released, Rover v0.26.3 is released, and various feature flags are flipped on our end). But it seemed reasonable to have an accurate changelog in the meantime. I'll send that to you once it's done, I have a draft.

Reply:
Wonderful, thanks!